main.py

```diff
@@ -82,11 +84,19 @@ async def signal_handler(sig):
                 self.dht = overlay
 
         # Initialize wallets
-        dummy_wallet1 = DummyWallet1()
-        self.wallets[dummy_wallet1.get_identifier()] = dummy_wallet1
-
-        dummy_wallet2 = DummyWallet2()
-        self.wallets[dummy_wallet2.get_identifier()] = dummy_wallet2
+        # dummy_wallet1 = DummyWallet1()
```

Comment:
How does this part work? Do you define all the currently available wallets here? Should we change this before we push it?

Reply:
I believe this was an accidental commit - I played around with this part of the code a lot for manual testing purposes. I don't see a point to changing it from its original state, that is, the way it was when the branch was initially created.